WapTenantPublicAPI.psm1

```diff
@@ -639,97 +645,107 @@ function Get-WAPVMRoleQuotaRequest{
         PS C:\>Get-WAPVMRoleQuotaRequest -Subscription $Sub -VMRoleSizeProfile (Get-WAPVMRoleVMSize -Name A7)
 
     #>
-Param(
-    [Parameter(
-        Mandatory=$True,
-        ValueFromPipeline=$True,
-        ValueFromPipelineByPropertyName=$True,
-        Position=0,
-        ParameterSetName = "Size-Object"
-    )]
-    [System.Management.Automation.PSTypeName('WAP.VMRoleSizeProfile')] 
-    $VMRoleSizeProfile,
-
-    [Parameter(
-        Mandatory=$True,
-        ValueFromPipeline=$True,
-        ValueFromPipelineByPropertyName=$True,
-        Position=0,
-        ParameterSetName = "Size-Name"
-    )]
-    [string] 
-    $Size,
+    [CmdletBinding( DefaultParameterSetName="Size-Name" )]
+    param(
+        [Parameter(
+            Mandatory,
+            ValueFromPipeline = $true,
+            ValueFromPipelineByPropertyName = $true,
+            ParameterSetName = "Size-Object"
```

Comment:
Some comments as above. If you use string literals (no variable expansion), please use single quotes.